ddtrace/tracer/span.go

```diff
@@ -30,6 +32,13 @@ var (
 	_ msgp.Decodable = (*spanLists)(nil)
 )
 
+// errorConfig is internal struct for passing the setTagError config.
```

Comment:
Generally, in Go, we should try and avoid pointing out the obvious in documentation (e.g. `internal` and `struct`) because we can see that it is a struct and that it is unexported. Maybe we can say something like:
```go
// errorConfig holds customization options for setting error tags.
```

Reply:
Fixed. Thanks!